configure

```diff
@@ -1989,6 +1993,25 @@ Available commands are:
       message.index("\n") != nil
     end
   end
+
+  # Returns true if MacPorts is installed in its default location.
+  def macports?
+    File.exists? '/opt/local/bin/port'
```

Comment:
While the canonical location might be `/opt/local/bin/port` we have no control over it, and we shouldn't enforce the user to _always_ place MacPorts in a fixed place. One option would be to default to the output of `which port` but allow users to specify a custom path (using something like `--macports=...`.